src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -14,26 +15,41 @@ interface IGenomicOverviewProps {
     sampleColors: {[s:string]:string};
     sampleManager: SampleManager;
 }
-export default class GenomicOverview extends React.Component<IGenomicOverviewProps, {vafPlotData:{[s:string]:number[]}}> {
+export default class GenomicOverview extends React.Component<IGenomicOverviewProps, { vafPlotData:{[s:string]:number[]}}> {
+
+    shouldComponentUpdate(){
+        return false;
+    }
 
     constructor(props:IGenomicOverviewProps) {
         super(props);
         const vafPlotData = this.computeVAFPlotData(props.mutations);
         this.state = {
             vafPlotData,
         };
+
+    }
+
+    componentDidMount(){
+
+        var debouncedResize =  _.debounce(()=>this.forceUpdate(),500);
+
+        $(window).resize(debouncedResize);
+
     }
 
     public render() {
 
+        const width = $('.tab-content').width()-140;
+
         const labels = _.reduce(this.props.sampleManager.samples, (result: any, sample: ClinicalDataBySampleId, i: number)=>{
             result[sample.id] = i + 1;
             return result;
         }, {});
 
         return (
             <div style={{ display:'flex', alignItems:'center'  }}>
-                <Tracks mutations={this.props.mutations} sampleManager={this.props.sampleManager} cnaSegments={this.props.cnaSegments} />
+                <Tracks mutations={this.props.mutations} key={Math.random()} sampleManager={this.props.sampleManager} width={width} cnaSegments={this.props.cnaSegments} />
```

Comment:
Setting to random key forces the component to remount every time, thus redrawing the visualization.